README.md

```diff
@@ -33,16 +33,46 @@ fmt.Println(lang)
 lang := enry.GetLanguage("foo.cpp", "<cpp-code>")
 ```
 
-Developmemt
+Development
 -----------
-*enry* re-uses parts of original [linguist](https://github.com/github/linguist) especially data in `languages.yml` to generate internal data structures. In oreder to update to latest upstream run
+*enry* re-uses parts of original [linguist](https://github.com/github/linguist) especially data in `languages.yml` to generate internal data structures. In order to update to latest upstream run
 
     make clean code-generate
 
 To run the tests
 
     make test
 
+Benchmark
+---------
+Enry's language detection has been compared  with Linguist's language detection. In order to do that, linguist's project directory [*linguist/samples*](https://github.com/github/linguist/tree/master/samples) was used as a set of files to run benchmarks against.
```

Comment:
Minor: Now there's a mix of titles with a blank line afterwards and titles without one. Make it consistent.